packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -80,13 +81,15 @@ export const ChatComposite = (props: ChatCompositeProps): JSX.Element => {
   return (
     <BaseComposite {...props}>
       <ChatAdapterProvider adapter={adapter}>
-        <ChatScreen
-          options={options}
-          onFetchAvatarPersonaData={onFetchAvatarPersonaData}
-          onRenderTypingIndicator={onRenderTypingIndicator}
-          onRenderMessage={onRenderMessage}
-          onFetchParticipantMenuItems={onFetchParticipantMenuItems}
-        />
+        <div className={chatScreenContainerStyle}>
```

Comment:
Is it possible to have the div outside of the BaseComposite?

By changing the order then we can share that to get everything working you need to just drop this composite in and then customers can feel confident they don't need to control the dimensions from inside of the composite.

Reply:
I will need to change this for Call composite too then :)